DOCS.md

```diff
@@ -412,6 +413,18 @@ login({appState: JSON.parse(fs.readFileSync('appstate.json', 'utf8'))}, (err, ap
 
 ---------------------------------------
 
+<a name="forwardAttachment"></a>
+### api.forwardAttachment(attachmentID, userOrUsers[, callback])
+
+Takes a userID or an array of userIDs and forward the corresponding attachment.
+
+__Arguments__
+* `attachmentID`: A attachment ID (usualy `attribute.ID` property)
```

Comment:
Hey, you're doing great! It's a tricky thing to get perfect.
(Don't worry, @bsansouci's first language is French so he understands how you feel)